teku/src/test/java/tech/pegasys/teku/cli/options/P2POptionsTest.java

```diff
@@ -43,6 +50,7 @@
 public class P2POptionsTest extends AbstractBeaconNodeCommandTest {
 
   @Test
+  @Disabled("Temporarily disabled due to config file path issues in CI")
```

Comment:
Not sure why we are disabling this test?

Reply:
hrm I thought i had fixed that, will check if it didnt push